rpcs3/Emu/RSX/GL/GLGSRender.cpp

```diff
@@ -1024,7 +1024,7 @@ void GLGSRender::EnableVertexData(bool indexed_draw)
 
 			glEnableVertexAttribArray(i);
 			checkForGlError("glEnableVertexAttribArray");
-			glVertexAttribPointer(i, m_vertex_data[i].size, gltype, normalized, 0, reinterpret_cast<void*>(offset_list[i]));
+			glVertexAttribPointer(i, m_vertex_data[i].size, gltype, normalized, 0, reinterpret_cast<void*>((u64)offset_list[i]));
```

Comment:
My point being that it won't actually cause erroneous behavior and you're casting to do a cast. If you want to get rid of the warning I'd say just replace `reinterpret_cast<void*>((u64)` with straight up `(void *)`, since you're already doing a c-style cast.